packages/evolution-frontend/src/components/survey/Question.tsx

```diff
@@ -392,7 +392,15 @@ export class Question<CustomSurvey, CustomHousehold, CustomHome, CustomPerson> e
                     )}
                 </InputWidgetWrapper>
             </div>
-        );
+        )];
+
+        if ((widgetConfig as any).joinPrevious) {
```

Comment:
Ok, so it is a visual hack to display the widgets together

Reply:
The best way would be to create nested arrays for the widgets in the section config instead of a simple array, but it may be more difficult to implement